src/python/pants/backend/codegen/protobuf/go/subsystem.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from typing import cast
+
+from pants.option.subsystem import Subsystem
+
+
+class GoProtobufSubsystem(Subsystem):
+    options_scope = "go-protobuf"
+    help = (
+        "Go protobuf generator (https://pkg.go.dev/google.golang.org/protobuf/cmd/protoc-gen-go)."
+    )
+
+    @classmethod
+    def register_options(cls, register):
+        super().register_options(register)
+        register(
+            "--version",
+            type=str,
+            default="v1.27.1",
+            help=(
+                "The version of the Go protobuf plugin to use. The value of this option is used as "
```

Comment:
Nit, would be good to use `\n\n` to have second and third sentences be their own paragraph.

Reply:
done